src/pages/index.js

```diff
@@ -29,51 +43,124 @@ const renderCard = (item) => {
 };
 
 //create objects
+//api
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "7ff811c0-9f09-42c2-975e-cca434c12992",
+    "content-type": "application/json",
+    "Access-Control-Allow-Origin": "*",
+  },
+});
+
 //profile section
-const userInformation = new UserInfo(".profile");
+const userInfo = new UserInfo(".profile");
+
+const profileSection = new Section(
+  {
+    renderer: () => {
+      api.getUserInformation().then(({ name, avatar, about, _id }) => {
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all` not to make nested requests 

More info is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

```js

Promise.all([getUserInfo(), getCards()])
// destructure the response
  .then(([userData, cards]) => {
      // set all the data
  })
  .catch(err => {
    // catch possible errors
  });

```